contrib/cirrus/win-collect-wsl-logs-stop.ps1

```diff
@@ -0,0 +1,23 @@
+#Requires -RunAsAdministrator
+
+# This script has been copied from the WSL repo:
+# https://github.com/Microsoft/WSL/blob/master/diagnostics/collect-wsl-logs.ps1
```

Comment:
This file and the other copied ones are MIT licensed. We should probably reproduce the license here to make it clear it's not under the same Apache2 as the rest of the repo.